CHANGELOG.md

```diff
@@ -10,12 +10,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]
 ------------
 
+[v0.7.0-rc1] - 2019-10-08
+-------------------------
+
+### Added
+
+- Support Intel DCAP attestation on Windows.
+- Support enclave debugging on Windows with WinDBG/CDB.
+    - The new oedebugrt.dll binary needs to be copied to the app folder to enable this.
+- Support `transition_using_threads` EDL attribute in oeedger8r.
+    - This only applies to untrusted functions (ocalls) in this release.
+    - Using this attribute allows the ocall to be invoked without incurring the performance cost of an enclave context switch.
+
 ### Changed
 
 - Transferred repository from [microsoft/openenclave](https://github.com/microsoft/openenclave) to [openenclave/openenclave](https://github.com/openenclave/openenclave).
 - Change debugging contract for oegdb. Enclaves and hosts built prior to this release cannot be debugged with this version of oegdb and vice versa.
+- Update Intel DCAP library dependencies to 1.2.
+- Update Intel PSW dependencies to 2.6 on Linux and 2.4 on Windows.
 - Update LLVM libcxx to version 8.0.0.
-- Update mbedTLS to version 2.7.11.
+- Update mbedTLS to version 2.16.2.
+
```

Comment:
This looks good. Thanks for getting these changes in @CodeMonkeyLeet .
We can consider adding that MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_KEY_EXCHANGE and MBEDTLS_KEY_EXCHANGE_RSA_ENABLED will be deprecated.

Reply:
Added as per #2038